commit d08d4142ec75d931b364a002c8eaae91223f1ac5
Author: Anant Kamath <kamathanant@gmail.com>
Date:   Sat Dec 7 19:06:33 2013 +0530

    Build pintxo plugin only if MMQt 0.5.1 is found, higher/lower versions break the plugin

diff --git a/plugins/pintxo/CMakeLists.txt b/plugins/pintxo/CMakeLists.txt
index aeb5b62..ba74b84 100644
--- a/plugins/pintxo/CMakeLists.txt
+++ b/plugins/pintxo/CMakeLists.txt
@@ -2,7 +2,7 @@ project (ktpaccountskcm-plugin-pintxo)
 
 INCLUDE(FindPkgConfig)
 
-pkg_check_modules(LIBMODEMMANAGERQT ModemManagerQt>=0.5.1)
+pkg_check_modules(LIBMODEMMANAGERQT ModemManagerQt=0.5.1)
 if (LIBMODEMMANAGERQT_FOUND)
     MESSAGE("ModemManagerQt >= 0.5.1 found, building pintxo plugin for SMS messaging")
     include_directories (${CMAKE_CURRENT_BINARY_DIR}
